packages/cli/lib/conflicter.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
```

Comment:
Why do we need to have our own conflicter?

Reply:
Issue https://github.com/strongloop/loopback-next/issues/738 specified that the CLI should not allow overwriting of an existing controller with the same name, and the existing Conflicter does not allow the specification of default action to take when file name conflicts come up. I think through Conflicters are the only way we can meet the requirement, unless the requirement itself is flexible.

Reply:
By default, Yeoman will prompt you if there is a conflict. I think that's good enough. Another options is to check the existence of the file directly using `fs`. The `app` or `extension` generator does so to make sure the target dir is not empty. See https://github.com/strongloop/loopback-next/blob/master/packages/cli/lib/project-generator.js#L113